ocfweb/static/scss/pages/_tv.scss

```diff
@@ -2,17 +2,26 @@ $base-font-size: 6vw;
 $tv-hours-large: 12;
 $tv-hours-small: 5;
 
-.page-tv {
+.page-tv-hours {
 
     background-image: url('../img/tv/background.png');
     background-attachment: fixed;
     background-repeat: no-repeat;
+    background-position: center center;
     background-size: cover;
-    height: 100vh;
+}
+
+.page-tv-body {
```

Comment:
Here's a slightly easier way to do this, that we can make use of since we're using Sass:

```css
.page-tv-hours {
    /* Stuff that gets applied to everything on this page goes here */
    ...

    body {
        /* Stuff that gets applied to the <body> element and below goes here, e.g.: */
        background-image: url('../img/tv/background.png');
        background-attachment: fixed;
        ...
    }
}
```

Then you won't separately need a `.page-tv-body` class anymore.

See https://github.com/ocf/ocfweb/blob/ea795318a1b19298f03080ff2c6cc9c7d1322b84/ocfweb/static/scss/pages/about/_lab-open-source.scss#L16 for an example.